src/common/snippets/src/lowered/pass/allocate_buffers.cpp

```diff
@@ -57,34 +58,40 @@ void AllocateBuffers::propagate_offset(const LinearIR& linear_ir, const Expressi
 
 bool AllocateBuffers::run(LinearIR& linear_ir) {
     OV_ITT_SCOPED_TASK(ov::pass::itt::domains::SnippetsTransform, "Snippets::AllocateBuffers");
+    // [113664] The pass contains two main logics: it defines which of buffers can be inplace (use the same memory) and
+    // allocate memory of needed size. It should be splitted into several passes and updated in bounds of the ticket 113664.
 
-    size_t offset = 0;
+    size_t offset = 0, wsp_offset = 0;
+    size_t prev_data_size = 0, current_data_size = 0;
     std::set<ExpressionPtr> allocated_buffers;
+    bool wsp_allocated = false;
```

Comment:
Minor: could you please provide explanation on what `wsp` means? Please add a comment, or better provide a self-explanatory names to `wsp_offset` and `wsp_allocated`

Reply:
This name is used in `Brgemm` AMX case. I renamed the variable and added the comment with the ticket. I believe we will rewrite Buffer logic asap 🙂 
Thank you